doc/classes/Button.xml

```diff
@@ -58,7 +58,7 @@
 			To edit margin and spacing of the icon, use [theme_item h_separation] theme property and [code]content_margin_*[/code] properties of the used [StyleBox]es.
 		</member>
 		<member name="icon_alignment" type="int" setter="set_icon_alignment" getter="get_icon_alignment" enum="HorizontalAlignment" default="0">
```

Comment:
I guess a better name is `horizontal_icon_alignment`, but to keep backwards compatibility it is better to not touch it.
Still maybe an idea in the future.